be/Cf.WebApi/Program.cs

```diff
@@ -41,6 +41,12 @@
 
 app.UseHttpsRedirection();
 
+var MyAllowSpecificOrigins = "_myAllowSpecificOrigins";

```

Comment:
this looks like it should belong in app settings. Going to provide a similar scenario by Matas V how to use it.
![image](https://github.com/MatasAdmoneo/CarFans/assets/120173596/400250da-0618-46b7-8aaf-6cf8e63e61de)
![image](https://github.com/MatasAdmoneo/CarFans/assets/120173596/45d0f32e-ef6b-41b9-b635-c423e9571145)

also `configuration` is already defined in `Program.cs` file so don't assign a new one

Reply:
`var origins =  configuration["SomeName:SomeName"];`